backend/src/main/java/co/inventorsoft/scripty/model/entity/Project.java

```diff
@@ -43,5 +45,8 @@
 	User user;
 	
 	LocalDateTime createDate;
+
+	@Lob
+	DirectoryNode filesMetadata;
```

Comment:
How will this field be serialized? As far as I can see it will use standard Java serialization. Please, add attribute converter which will serialize it as json

Reply:
I've tried. But attribute converter via json has a problem with deserialization. It doesn't understand polymorphism. 

"response": "Unavailable to read data from database.\nUnrecognized field \"children\" (class co.inventorsoft.scripty.model.dto.Node), not marked as ignorable (4 known properties: \"parent\", \"name\", \"path\", \"type\"])\n at [Source: (StringReader); line: 1, column: 152] (through reference chain: co.inventorsoft.scripty.model.dto.DirectoryNode[\"children\"]->java.util.LinkedList[0]->co.inventorsoft.scripty.model.dto.Node[\"children\"])"

Standard Java serialization works correctly.

Reply:
Jackson supports polimorphic deserialization. He just need a bit of help from you. Take a look at this answer [https://stackoverflow.com/questions/30362446/deserialize-json-with-jackson-into-polymorphic-types-a-complete-example-is-giv](https://stackoverflow.com/questions/30362446/deserialize-json-with-jackson-into-polymorphic-types-a-complete-example-is-giv)